yoti-sdk-api/src/main/java/com/yoti/api/client/DigitalIdentityClient.java

```diff
@@ -34,45 +38,34 @@ public class DigitalIdentityClient {
     /**
      * Create a sharing session to initiate a sharing process based on a policy
      *
-     * @param request
-     *              Details of the request like policy, extensions and push notification for the application
-     * @return an {@link ShareSession}
-     *              Id, status and expiry of the newly created Share Session
-     * @throws DigitalIdentityException
-     *              Aggregate exception signalling issues during the call
+     * @param request Details of the request like policy, extensions and push notification for the application
+     * @return an {@link ShareSession} ID, status and expiry of the newly created Share Session
+     * @throws DigitalIdentityException Thrown if the session creation is unsuccessful
```

Comment:
There should be consistency in the formatting of the Javadocs... either always a newline + identation for descriptions or never. But in some places doing it and in others not doesn't look good.

Also FYI, the first sentence of a javadoc should end with a full stop `.`